plugins/lime-plugin-fbw/src/containers/NetworkForm.js

```diff
@@ -36,20 +27,18 @@ export const NetworkForm = ({ createNetwork, toggleForm }) => {
 		return e;
 	}
 
-	function _changeHostName(e) {
-		const end = e.type === 'change';
-		e.target.value = slugify(e.target.value, end);
-		setState({ ...state, hostName: e.target.value || '' });
-		return e;
-	}
-
```

Comment:
this function seems to have been moved untouched just a few lines down.
Would leave it were it is to simplify the commit.